static/src/javascripts-legacy/projects/commercial/modules/third-party-tags.js

```diff
@@ -60,9 +60,7 @@ define([
             .then(init);
         }
 
-        var isMobileOrTablet = ['mobile', 'tablet'].indexOf(detect.getBreakpoint(false)) >= 0;
-        var shouldIgnoreSwitch =  isMobileOrTablet || config.page.section === 'world' || config.page.edition.toLowerCase() !== 'au';
-        var shouldServePlista = config.switches.plistaForOutbrainAu && !shouldIgnoreSwitch;
+        var shouldServePlista = config.switches.plistaForOutbrainAu && config.page.edition.toLowerCase() === 'au';
```

Comment:
True - I think I'll leave it for now as this may all change in the near future anyway given the current Outbrain tests.